### DISCUSSION TITLE
Reason Rack Plugin

### DISCUSSION THREAD

**Discussion by MRKLAB** [apohl79/audiogridder#612] [Help]
Hi all !
I'm having problem (a big problem) using Reason Rack Plugin , it works fine but when i close and reload the project it shows the default blank rack , no device loaded , instruments , effects or whatelse.

Cubase Pro 9.5 on w7 + AG 1.1 beta 10 on w8.1 , all legit sw.
With the same config I've no problem with other plugins.

**Comment by CaBuffalo** [comment]
Hi,there:

It looks like you are using an old version. Can you try the new 1.2 Beta 3
or 1.1 stable ?

Thanks.

On Tue, Jun 15, 2021 at 1:34 AM MRKLAB ***@***.***> wrote:

> Hi all !
> I'm having problem (a big problem) using Reason Rack Plugin , it works
> fine but when i close and reload the project it shows the default blank
> rack , no device loaded , instruments , effects or whatelse.
>
> AG 1.1 beta 10 on w8.1 , all legit sw.
> With the same config I've no problem with other plugins.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/612>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6H4OOIEBAFEWZWOFPLTS4GCJANCNFSM46WYZX6A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by MRKLAB** [reply]
Yes @CaBuffalo (and thnx for the reply) , i,'m still using the old beta release because it's very stable and usable on my sistem apart from this prob.
Well , then i give a try to the new beta (v1.2) because i suppose that the v1.1 stable don't differ a lot from the v1.1 beta10 , right ?
I check and report asap.
Thanks !

**Reply by MRKLAB** [reply]
@CaBuffalo , tryed but nope , it doesn't works !
Also , if i change the latency value of the soundcard (RME Digiface) on my daw (due to project grown for example) Reason Rack Plugin reset to its default blank status.
Now , the one billion $ question is , could be a Reason Rack Plugin problem instead of an AG issue ?

**Reply by apohl79** [reply]
Can you try to set the sync to "every 10 seconds" and see if that helps?

![image](https://user-images.githubusercontent.com/1129160/122182085-3c745e00-ce8a-11eb-8799-9444cf174132.png)

**Reply by MRKLAB** [reply]
Thanks @ apohl79, tried but again doesn't works.
And more, the pan manager change in this way when i load AG.
![A](https://user-images.githubusercontent.com/75311025/122187028-f1107e80-ce8e-11eb-96fa-799adcb242dc.JPG)

**Comment by MRKLAB** [comment]
UP !
